package.json

```diff
@@ -75,6 +75,7 @@
     "socket.io-client": "1.7.3",
     "stylelint": "7.9.0",
     "urijs": "1.18.9",
-    "webpack": "2.2.1"
+    "webpack": "2.2.1",
+    "webpack-remove-debug": "0.1.0"
```

Comment:
I tried to inline it (using a function) but it looks like we have to make it an internal module in its own file and set it as `loader: "./file"`. That sucks. Anyone knows a way we could use to simply define this one-line function at the top of the webpack config and call that function from the loader?!